openapi/paths/credit-memos@{id}@transaction-allocations@{transactionId}.yaml

```
@@ -0,0 +1,97 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: transactionId
+    in: path
+    description: Transaction ID.
+    required: true
+    schema:
+      type: string
+get:
+  x-products:
+    - Core
+  tags:
+    - Credit memos
+  summary: Get amount allocated from a credit memo to a transaction
+  operationId: GetCreditMemoTransactionAllocation
+  x-sdk-operation-name: getCreditMemoTransactionAllocation
+  description: >
+    Gets the amount allocated from the given credit memo to the given transaction.
```

nitpick: I think we should start getter descriptions with "Retrieves"

```suggestion
    Retrieves the amount allocated from the given credit memo to the given transaction.
```